src/app/shared/entry/verified-by/verified-by.component.ts

```diff
@@ -15,7 +15,7 @@ export class VerifiedByComponent implements OnChanges {
 
   ngOnChanges() {
     if (this.version) {
-    this.verifiedByStringArray = this.verifiedByService.getVerifiedByString(this.version.sourceFiles);
+      this.verifiedByStringArray = this.verifiedByService.getVerifiedByString(this.version.sourceFiles);
```

Comment:
I don't think this will ever happen nor will it be an issue if it does, but just checking... can version ever get unset after it has been set? If so, then `this.verifiedByStringArray` wouldn't be cleared out.